source/includes/steps-starting-compass.yaml

```diff
@@ -60,9 +67,12 @@ content: |
    the :guilabel:`Connect` button to navigate to the |compass-short|
```

Comment:
I just realized that we don't have a lead-in sentence above the tab set? I'd be in favor of adding one, though I am concerned that the TIP will squash it. Thoughts?

Reply:
I don't think it looks too squashed, but lmk what you think.

Reply:
Looks good!